src/components/List/List.tsx

```diff
@@ -120,12 +119,40 @@ class List extends UIComponent<Extendable<ListProps>, FocusContainerState> {
     )
   }
 
+  componentDidMount() {
+    this.focusHandler = new ContainerFocusHandler(
+      () => this.props.items.length,
+      index => {
+        const targetComponent = this.itemRefs[index] && this.itemRefs[index].current
+        const targetDomNode = ReactDOM.findDOMNode(targetComponent) as any
+
+        targetDomNode && targetDomNode.focus()
+      },
+    )
+  }
+
   renderItems() {
     const { items, renderItem } = this.props
-    const itemProps = _.pick(this.props, List.itemProps)
 
+    this.itemRefs = []
     return _.map(items, (item, idx) => {
-      itemProps.focusableItemProps = this.focusContainer.createItemProps(idx, items.length)
+      let maybeSelectableItemProps = {}
+
+      if (this.props.selection) {
+        const _ref = React.createRef()
```

Comment:
some time ago we had a discussion in other PR when underscore was used for a local variable as here and we agreed that it should be removed :)

Reply:
ok, will remove